src/Data/Monoid/Additive.purs

```diff
@@ -0,0 +1,44 @@
+module Data.Monoid.Additive where
+
+import Prelude
```

Comment:
Shouldn't import itself :smile_cat: (similarly with the others files)

Reply:
These ones are ok, as the `Prelude` module doesn't re-export them :)

Reply:
Oh yes, of course!